### PR TITLE
2. Find generics

### PR DESCRIPTION
**Pull request by ZeeshaanMaudar**
### Work Done
Add the find generics feature.
Call to different api
Fix markup for semantic purposes
Add prop validations

### CODE REVIEW

File: src/components/MedicineBasicSearch/index.js [hunk 1]
```diff
@@ -28,29 +29,53 @@ const callName = name => (
...
         {callName(name)}
-        {callPriceAndGenerics(price)}
+        {callPriceAndGenerics(price, nappiCode, fetchGenerics)}
       </div>
     </div>
-  ))
+  );
+};
+
+const MedicineBasicSearch = ({ results, fetchGenerics }) => (
```
Comment (schalkventer) [inline src/components/MedicineBasicSearch/index.js:64]: Make single line

File: src/components/SearchTool/Markup.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import t from 'prop-types';
+import MedicineBasicSearch from '../MedicineBasicSearch';
+
+import classes from './style.module.css';
+
+const callForm = (submitForm, content, changeHandler) => (
+
```
Comment (schalkventer) [inline src/components/SearchTool/Markup.js]: Remove newline

File: src/components/SearchTool/Markup.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    results,
+    submitForm,
+    changeHandler,
+    fetchGenerics,
+  } = props;
+
+  return (
+    <React.Fragment>
+      {callForm(submitForm, content, changeHandler)}
+      <MedicineBasicSearch {...{ fetchGenerics }} results={results} />
```
Comment (schalkventer) [inline src/components/SearchTool/Markup.js]: Add results as well to destructuring

File: src/components/SearchTool/Markup.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    sep: t.string,
+    nappi_code: t.string,
+  })),
+  submitForm: t.func.isRequired,
+  changeHandler: t.func.isRequired,
+  fetchGenerics: t.func.isRequired,
+};
+
+Markup.defaultProps = {
+  content: '',
```
Comment (schalkventer) [inline src/components/SearchTool/Markup.js]: Set to `null`

File: src/components/MedicineBasicSearch/index.js (hunk 1 above)
Comment (ZeeshaanMaudar) [reply src/components/MedicineBasicSearch/index.js:64]: eslint complains that line is too long

**Review by schalkventer — APPROVED** [review]
👽

**Pull request merged**